db/seeds.rb

```diff
@@ -1,3 +1,7 @@
 # frozen_string_literal: true
 
-User.create!(email: 'test@test.com', password: 'password')
+User.create!(email: 'test@test.com', password: 'password',
```

Comment:
Yes. But I just Destroy all entries in before create, it is up to you how you will implement this.